cloud/blockstore/libs/storage/stats_service/stats_service_actor_solomon.cpp

```diff
@@ -310,11 +321,29 @@ void TStatsServiceActor::HandleVolumePartCounters(
 
     State.GetCounters(volume->VolumeInfo).UpdatePartCounters(*msg->DiskCounters);
 
+    const auto mediaKind = volume->VolumeInfo.GetStorageMediaKind();
+
+    if (volume->IsDiskRegistryBased() && mediaKind != STORAGE_MEDIA_SSD_LOCAL) {
+        auto& rdmaCounter = State.GetRdmaCounter(mediaKind).PartAcc.Rdma;
+        auto& interconnectCounter =
+            State.GetInterconnectCounter(mediaKind).PartAcc.Interconnect;
+
+        rdmaCounter.ReadBlocks.Add(msg->DiskCounters->Rdma.ReadBlocks);
+        rdmaCounter.WriteBlocks.Add(msg->DiskCounters->Rdma.WriteBlocks);
+        interconnectCounter.ReadBlocks.Add(
+            msg->DiskCounters->Interconnect.ReadBlocks);
+        interconnectCounter.WriteBlocks.Add(
+            msg->DiskCounters->Interconnect.WriteBlocks);
```

Comment:
why not call Publish() here instead of in the end of the function and for all counters instead of mediaKind specific

Reply:
>  for all counters instead of mediaKind specific


There are no transport counters for other mediaKinds, so trying to call GetInterconnectCounter() for this will result in an error:
https://github.com/ydb-platform/nbs/blob/6f365a30d75d49f4de5c755b0bca167086e6d56c/cloud/blockstore/libs/storage/stats_service/stats_service_state.h#L413